core/injector/application_injector.cpp

```diff
@@ -330,7 +330,8 @@ namespace {
       return initialized.value();
     }
     auto options = leveldb::Options{};
-    options.max_open_files = 1500;  // 1000 was the default value
+    options.max_open_files =
+        -1;  // Unlimited (fix for 'Too many open files' error)
```

Comment:
Looks like magic number. Seems is better to use numeric_limit. 
On other side, I guess special leveldb constant exists for that.

Reply:
I couldn't find any relevant constants within leveldb (the default 1000 is set as a magic number as well). But I will add our own for -1.